src/practice/ts/Components/Store/Add.ts

```diff
@@ -0,0 +1,45 @@
+import Storage from "./storage";
+
+class Add {
+  private formElem: HTMLFormElement;
+  private inputElem: HTMLInputElement;
+  private storageContainer: HTMLUListElement;
+  private storage: Storage;
+  private id: number;
+
+  constructor(
```

Comment:
인자가 여러개니까 객체로 받는 것도 고려해보시면 좋을 것 같아요. 순서가 헷갈릴것 같네요.
```suggestion
  constructor({...})
```

Reply:
완전 동의합니다. 수정사항에 반영했습니다.